pages/_app.tsx

```diff
@@ -75,6 +76,7 @@ export default function App({ Component, pageProps }: AppProps) {
                 <main>
                   <div className="relative z-10 h-full flex-grow">
                     <Toaster position="bottom-right" />
+                    <ChainChecker />
```

Comment:
Please refer to the comment I did to this component as it might not be necessary